src/lib/components/works_for_you/notification.tsx

```diff
@@ -11,19 +12,32 @@ import colors from "../../../data/colors"
 interface Props extends RelayProps {}
 
 export class Notification extends React.Component<Props, any> {
+  handleArtistTap() {
+    // Hacky workaround until artist href is provided through metaphysics
+    const artistHref = this.props.notification.artworks[0].artists[0].href
+    SwitchBoard.presentNavigationViewController(this, artistHref)
+  }
+
   render() {
     const notification = this.props.notification
 
+    // artwork-less notifications are rare but possible and very unsightly
```

Comment:
This will also be dealt with in metaphysics but at least for now emission can catch it and not render the artwork-less notifications.